src/_pytest/logging.py

```diff
@@ -420,6 +420,44 @@ def __init__(self, config):
 
         self.log_cli_handler = None
 
+        terminal_reporter = self._config.pluginmanager.get_plugin("terminalreporter")
```

Comment:
Should this code kept being wrapped in a function/method (`_setup_cli_logging`)?

Reply:
I also prefer keeping the _setup_cli_method. Thx for pointing that out!